src/js/tools/drawing/Lighten.js

```diff
@@ -65,9 +70,15 @@
       }
     }
     if (color) {
```

Comment:
After checking the code you can get rid of the check : tinycolor and tinycolor.darken/lighten always return a valid tinycolor instance.